packages/react-tinacms-github/src/github-client/index.ts

```diff
@@ -71,9 +74,31 @@ export class GithubClient {
         base: this.baseBranch,
       },
     })
+
+    this.setCookie(GithubClient.FORK_COOKIE_KEY, pullRequest.full_name)
```

Comment:
Did you mean to put this in the `createFork` function?

Reply:
Yes that's how it was being used before.

Reply:
But it was being done in [the modal](https://github.com/tinacms/tinacms/blob/master/packages/react-tinacms-github/src/github-editing-context/GithubAuthModal.tsx#L85-L86)

Reply:
Nope I'm tripping out